tests/openwisp2/sample_firmware_upgrader/admin.py

```diff
@@ -1,26 +1,5 @@
-from django.contrib import admin
-from openwisp_firmware_upgrader.base.admin import (
-    AbstractBatchUpgradeOperationAdmin,
-    AbstractBuildAdmin,
-    AbstractCategoryAdmin,
+from openwisp_firmware_upgrader.admin import (  # noqa
```

Comment:
I think we should mention in README to unregister and register a class again in case you want to make changes to an admin class, right?

Reply:
If they want to make big changes, yes, it would be better for them to unregister, extend the class and register it again, for small changes they can resort to monkey patching.

Reply:
I am thinking about further improving this, will push shortly.

Reply:
For a moment I thought that we could check if the main app is in `INSTALLED_APPS` and if not, we can avoid registering the admin classes.

But with the information I have now, the amount of people needing this is not high enough to justify the effort.

I think unregistering and registering the admin classes is good enough.

Reply:
One possible easy solution can be: 
```
example_model = swapper.load_model("example")
if not admin.is_registered(example_model):
    admin.register(example_adminclass, example_model)
```

Reply:
this where, in the main admin.py file?

Reply:
I tested it in openwisp_ipam, it wouldn't work! :-(